main.py

```diff
@@ -91,4 +96,5 @@ def display():
     return redirect('/')
 
 
```

Comment:
I had to do this so it didn't start your flask app server when we imported the get_output_error from main.py in our test!